mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -419,6 +421,12 @@ private CloseableIterable<T> newParquetIterable(
           if (nameMapping != null) {
             parquetReadBuilder.withNameMapping(NameMappingParser.fromJson(nameMapping));
           }
+          if (task.file().keyMetadata() != null) {
```

Comment:
Please apply the style guidelines for whitespace throughout.

Reply:
Added empty lines after each block in this PR, per the comment above. If other style guidelines are meant, please let me know.